dll/win32/iphlpapi/iphlpapi.spec

```diff
@@ -137,41 +137,22 @@
 @ stub SetTcpEntryToStack
 @ stdcall UnenableRouter( ptr ptr )
 
-# These are actually stubs, but we need to forward them to preserve the decoration.
-@ stdcall -arch=i386 _PfAddFiltersToInterface@24() _PfAddFiltersToInterface@24
-@ stdcall -arch=i386 _PfAddGlobalFilterToInterface@8() _PfAddGlobalFilterToInterface@8
-@ stdcall -arch=i386 _PfBindInterfaceToIPAddress@12() _PfBindInterfaceToIPAddress@12
-@ stdcall -arch=i386 _PfBindInterfaceToIndex@16() _PfBindInterfaceToIndex@16
-@ stdcall -arch=i386 _PfCreateInterface@24() _PfCreateInterface@24
-@ stdcall -arch=i386 _PfDeleteInterface@4() _PfDeleteInterface@4
-@ stdcall -arch=i386 _PfDeleteLog@0() _PfDeleteLog@0
-@ stdcall -arch=i386 _PfGetInterfaceStatistics@16() _PfGetInterfaceStatistics@16
-@ stdcall -arch=i386 _PfMakeLog@4() _PfMakeLog@4
-@ stdcall -arch=i386 _PfRebindFilters@8() _PfRebindFilters@8
-@ stdcall -arch=i386 _PfRemoveFilterHandles@12() _PfRemoveFilterHandles@12
-@ stdcall -arch=i386 _PfRemoveFiltersFromInterface@20() _PfRemoveFiltersFromInterface@20
-@ stdcall -arch=i386 _PfRemoveGlobalFilterFromInterface@8() _PfRemoveGlobalFilterFromInterface@8
-@ stdcall -arch=i386 _PfSetLogBuffer@28() _PfSetLogBuffer@28
-@ stdcall -arch=i386 _PfTestPacket@20() _PfTestPacket@20
-@ stdcall -arch=i386 _PfUnBindInterface@4() _PfUnBindInterface@4
-
-# x64 does not use decoration in these names
-@ stdcall -arch=x86_64 _PfAddFiltersToInterface@24()
-@ stdcall -arch=x86_64 _PfAddGlobalFilterToInterface@8()
-@ stdcall -arch=x86_64 _PfBindInterfaceToIPAddress@12()
-@ stdcall -arch=x86_64 _PfBindInterfaceToIndex@16()
-@ stdcall -arch=x86_64 _PfCreateInterface@24()
-@ stdcall -arch=x86_64 _PfDeleteInterface@4()
-@ stdcall -arch=x86_64 _PfDeleteLog@0()
-@ stdcall -arch=x86_64 _PfGetInterfaceStatistics@16()
-@ stdcall -arch=x86_64 _PfMakeLog@4()
-@ stdcall -arch=x86_64 _PfRebindFilters@8()
-@ stdcall -arch=x86_64 _PfRemoveFilterHandles@12()
-@ stdcall -arch=x86_64 _PfRemoveFiltersFromInterface@20()
-@ stdcall -arch=x86_64 _PfRemoveGlobalFilterFromInterface@8()
-@ stdcall -arch=x86_64 _PfSetLogBuffer@28()
-@ stdcall -arch=x86_64 _PfTestPacket@20()
-@ stdcall -arch=x86_64 _PfUnBindInterface@4()
+@ stdcall _PfAddFiltersToInterface@24()
```

Comment:
With this changed, the gcc build of iphlpapi seems to be missing the leading underscores (but has the @24 etc). It looks okay in master.

Reply:
This is what I get:
![grafik](https://user-images.githubusercontent.com/313067/64037866-f932e900-cb56-11e9-816a-abaf7c3212ff.png)

Reply:
Hah. We're both right. Seemingly it exports each function with both names.
```
>dumpbin/exports ..\output-MinGW-i386\dll\win32\iphlpapi\iphlpapi.dll | findstr "Pf"
        119   76 0000E3EC PfAddFiltersToInterface@24
        120   77 0000E448 PfAddGlobalFilterToInterface@8
        121   78 0000E4A4 PfBindInterfaceToIPAddress@12
        122   79 0000E500 PfBindInterfaceToIndex@16
        123   7A 0000E55C PfCreateInterface@24
        124   7B 0000E5B8 PfDeleteInterface@4
        125   7C 0000E614 PfDeleteLog@0
        126   7D 0000E66C PfGetInterfaceStatistics@16
        127   7E 0000E6C8 PfMakeLog@4
        128   7F 0000E724 PfRebindFilters@8
        129   80 0000E780 PfRemoveFilterHandles@12
        130   81 0000E7DC PfRemoveFiltersFromInterface@20
        131   82 0000E838 PfRemoveGlobalFilterFromInterface@8
        132   83 0000E894 PfSetLogBuffer@28
        133   84 0000E8F0 PfTestPacket@20
        134   85 0000E94C PfUnBindInterface@4
        155   9A 0000E3EC _PfAddFiltersToInterface@24
        156   9B 0000E448 _PfAddGlobalFilterToInterface@8
        157   9C 0000E4A4 _PfBindInterfaceToIPAddress@12
        158   9D 0000E500 _PfBindInterfaceToIndex@16
        159   9E 0000E55C _PfCreateInterface@24
        160   9F 0000E5B8 _PfDeleteInterface@4
        161   A0 0000E614 _PfDeleteLog@0
        162   A1 0000E66C _PfGetInterfaceStatistics@16
        163   A2 0000E6C8 _PfMakeLog@4
        164   A3 0000E724 _PfRebindFilters@8
        165   A4 0000E780 _PfRemoveFilterHandles@12
        166   A5 0000E7DC _PfRemoveFiltersFromInterface@20
        167   A6 0000E838 _PfRemoveGlobalFilterFromInterface@8
        168   A7 0000E894 _PfSetLogBuffer@28
        169   A8 0000E8F0 _PfTestPacket@20
        170   A9 0000E94C _PfUnBindInterface@4
```